src/models/disassemblyoutput.cpp

```diff
@@ -119,6 +119,52 @@ QString findBinaryForSymbol(const QStringList& debugPaths, const QStringList& ex
 
     return {};
 }
+
+int longestCommonLengthFromEnd(const QString& a, const QString& b)
+{
+    const auto lengthA = a.length();
+    const auto lengthB = b.length();
+    int shortestLength = std::min(lengthA, lengthB);
```

Comment:
`const auto`

...but:

I believe this whole thing is basically an application of std::mismatch, no? see

https://en.cppreference.com/w/cpp/algorithm/mismatch

Reply:
I was sure something like this was in the standard, but I could find anything. Seems like I used the wrong search words